app/forms/UploadSupportingMaterialMultipleFormProvider.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import javax.inject.Inject
+
+import forms.mappings.Mappings
+import play.api.data.Form
+
+class UploadSupportingMaterialMultipleFormProvider @Inject() extends Mappings {
+
+  def apply(): Form[String] =
+    Form(
+      "value" -> text("uploadSupportingMaterialMultiple.error.required")
+        .verifying(maxLength(100, "uploadSupportingMaterialMultiple.error.length"))
```

Comment:
Is this not a file?

Reply:
Or is there an input box too

Reply:
This page is a placeholder and there is a separate story for this page.